compiler/src/scala/tools/nsc/backend/ScalaPrimitivesOps.scala

```diff
@@ -0,0 +1,238 @@
+/* NSC -- new Scala compiler
+ * Copyright 2005-2013 LAMP/EPFL
+ * @author  Martin Odersky
+ */
+
+package scala
+package tools.nsc
```

Comment:
While you're at it, you could also change the packages from scala.tools.nsc to dotty.tools.dotc, this way we can get both scalac and dotty on the same classpath without clashes.